sdk/eventhub/Azure.Messaging.EventHubs/tests/Diagnostics/DiagnosticsTests.cs

```diff
@@ -0,0 +1,152 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
```

Comment:
This should replace the existing tests that were brought forward to mirror the Track One diagnostics, right?  It's not intended to supplement?

Reply:
Correct.

Reply:
Good deal, I'll make them go away after this gets in, then.